client/src/locale/nl.js

```diff
@@ -19,7 +19,7 @@ const nl = {
         info: [
             //Arrays of titles and info blocks and if a function is an admin function
             ["Uitnodigen", "<p>Instellingsbeheerders kunnen rollen aanmaken voor hun applicaties.</p>" +
-            "<p>Applicaties zijn aan SURFconext gekoppelde diensten.</p>", true],
+            "<p>Applicaties zijn aan SURFconext gekoppelde applicaties.</p>", true],
```

Comment:
Dit wordt nu een wat cyclische zin die niks toevoegt. Kunnen we hier niet wel "diensten" behouden of een ander synoniem? "Applicaties" is hier dan nog steeds gebruikt als de leidende term, "diensten" als een omschrijving. Andere oplossing ook goed.

Reply:
Eens, de nieuwe zin is niet echt verhelderend. Het is even de vraag wat we hiermee willen zeggen. Ik denk dat we hiermee willen aangeven dat de beheerders naar SURFconext (straks Access) dashboards toe moeten om iets aan te passen aan applicatielijsten die ze te zien krijgen.

Ik zou dan iets schrijven als: 
Instellingsbeheerders kunnen rollen aanmaken voor hun applicaties. 
De applicatielijst bestaat uit applicaties die zijn gekoppeld aan SURFconext.

Maar dit is misschien wat te lang?